src/main/java/commands/Command.java

```diff
@@ -0,0 +1,30 @@
+package commands;
+
+import functions.*;
+import java.io.IOException;
+
+/**
+ * The base abstract class for different types of commands.
+ */
+public abstract class Command {
```

Comment:
Great job creating an abstract class Command to manage all existing commands which makes the code more readable!